front/package.json

```diff
@@ -84,6 +85,7 @@
     "not op_mini all"
   ],
   "devDependencies": {
+    "gh-pages": "^1.1.0",
```

Comment:
What's this for? We currently don't use github pages for hosting, since the app requires a proxy server to do the requests without cors.

Reply:
We tried to host it ourselves using this package. No longer need so I will remove it from dependencies.